openpype/hosts/blender/api/pipeline.py

```diff
@@ -216,6 +230,34 @@ def _discover_gui() -> Optional[Callable]:
     return None
 
 
+def is_work_file_out_of_date() -> bool:
```

Comment:
I think it's better to articulate concepts in a positive way, keep "workfile" a single word and explicitly say this is a function which "checks" things, not simply a boolean: 
```suggestion
def check_workfile_up_to_date() -> bool:
```

Therefore you'll need to reverse your logic.